From df56b20377639fc0ca70d0de7a94f30465af9e7f Mon Sep 17 00:00:00 2001
From: maximilian attems <max@stro.at>
Date: Tue, 16 Mar 2010 23:09:10 +0100
Subject: [PATCH] prevent nouveau from udev auto loading

we need working xserver-xorg-video-nouveau to ship it,
also maybe newer nouveau.
---
 drivers/gpu/drm/nouveau/nouveau_drv.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/nouveau/nouveau_drv.c b/drivers/gpu/drm/nouveau/nouveau_drv.c
index da3b93b..2d85e31 100644
--- a/drivers/gpu/drm/nouveau/nouveau_drv.c
+++ b/drivers/gpu/drm/nouveau/nouveau_drv.c
@@ -104,6 +104,7 @@ module_param_named(fbpercrtc, nouveau_fbpercrtc, int, 0400);
 #endif
 
 static struct pci_device_id pciidlist[] = {
+/* no autoload until xserver-xorg-video-nouveau
 	{
 		PCI_DEVICE(PCI_VENDOR_ID_NVIDIA, PCI_ANY_ID),
 		.class = PCI_BASE_CLASS_DISPLAY << 16,
@@ -115,6 +116,7 @@ static struct pci_device_id pciidlist[] = {
 		.class_mask  = 0xff << 16,
 	},
 	{}
+*/
 };
 
 MODULE_DEVICE_TABLE(pci, pciidlist);
-- 
1.5.6.5

